src/jobs.rs

```diff
@@ -359,6 +361,12 @@ impl CommitJobInstance {
         }
     }
 
+    fn get_main_build_ref_name (build_refs: &Vec<models::BuildRef>) -> &str {
```

Comment:
Should be a slice instead of a &Vec no?

Reply:
I'm not sure, should it? What would be the benefit of using a slice in this case?

I'm definitely not an expert in Rust, so I'm open to any reasonable suggestions.

Reply:
See https://rust-lang.github.io/rust-clippy/stable/index.html#ptr_arg